modules/nw-exposing-service.adoc

```diff
@@ -72,17 +72,41 @@ HTTP/1.1 200 OK
 
 endif::nodeport[]
 ifdef::nodeport[]
-. To expose a node port for the application, enter the following command. {product-title} automatically selects an available port in the `30000-32767` range.
+. To expose a node port for the application, modify the custom resource definition (CRD) of a service by entering by the following command:
 +
 [source,terminal]
 ----
-$ oc expose service nodejs-ex  --type=NodePort --name=nodejs-ex-nodeport --generator="service/v2"
+$ oc edit svc <service_name>
+----
++
+.Example output
+[source,yaml]
+----
+spec:
+  ports:
+  - name: 8443-tcp
+    nodePort: 30327 <1>
+    port: 8443
+    protocol: TCP
+    targetPort: 8443
+  sessionAffinity: None
+  type: NodePort <2>
+----
+<1> Optional: Specify the node port range for the application. {product-title} automatically selects an available port in the `30000-32767` range.
+<2> Defines the service type.
+
+. Verify that the service node port is updated with the range `30327`:
```

Comment:
```suggestion
. Verify that the service node port is updated with a port in the correct range, for example, `30327`:
```

Reply:
Suggesting this addition because it's automatically selecting a port, which could be something other than 30327. I think it just makes it a bit more clear. ("correct" may not be the best word here - maybe "appropriate" or "valid" would be better)

Reply:
Thank you, Kerry. 
Instead of saying the "correct" range, IMO we can use the actual range here- `30000-32767`. So updated with the range `30000-32767`.

Reply:
Thanks for pointing that out